app/src/androidTest/java/org/mozilla/fenix/ui/robots/NavigationToolbarRobot.kt

```diff
@@ -29,7 +29,7 @@ class NavigationToolbarRobot {
         val mDevice = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
```

Comment:
(unrelated to this file)

It felt a little jarring to me when I changed my toolbar setting to top, then went back to the home screen and saw it was still on the bottom.  @apbitner do we have any plans to move the home toolbar as well?

Reply:
Not at the moment. On the home screen we recently moved the search bar from the middle of the screen to the bottom but we can revisit in the future now that we have the dynamic toolbar.